pallets/protocol-fee/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+//! Runtime API definition for the protocol fee module.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Decode, Encode};
+use frame_support::dispatch::Vec;
+use polymesh_runtime_common::protocol_fee::ProtocolOp;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_runtime::traits::{SaturatedConversion, UniqueSaturatedInto};
+
+/// A capped version of `Balance` which is normally a `u128`, fit into `u64` which is a serializable
+/// type unlike `u128`. There are no fees that would not fit into `u64`.
+#[derive(Eq, PartialEq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
+#[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
+pub struct CappedFee(pub u64);
```

Comment:
could you please add the rpc json in the schema

Reply:
Added.